mono/tests/Makefile.am

```diff
@@ -444,6 +476,15 @@ BASE_TEST_CS_SRC=		\
 	pinvoke_ppcd.cs	\
 	bug-29585.cs
 
+if INSTALL_MOBILE_STATIC
+BASE_TEST_CS_SRC= \
+	$(BASE_TEST_CS_SRC_UNIVERSAL)
+else
+BASE_TEST_CS_SRC= \
+	$(BASE_TEST_MOBILE_STATIC_BLACKLIST) \
```

Comment:
now that I see it here, better call it `BASE_TEST_CS_SRC_MOBILE_STATIC_NOT_SUPPORTED` to make it consistent with the other variables which use the  BASE_TEST_CS_SRC_ prefix